src/main/java/org/nebula/contrib/ngbatis/utils/ResultSetUtil.java

```diff
@@ -260,6 +260,9 @@ public static boolean isMatchedErrorCode(ResultSet resultSet, ErrorCode... error
    * @return 是否命中session失效情况
    */
   public static boolean isSessionError(ResultSet resultSet) {
+    if (resultSet == null) {
+      return true;
+    }

```

Comment:
@soul-gin 
When executeWithParameter throw Exception sometimes cause by session error, 
and resultSet will be null at that moment.